src/d/actor/d_a_wbird.cpp

```diff
@@ -5,70 +5,208 @@
+    sp18.z = cM_scos(envLight.mWind.mTactWindAngleX) * cM_ssin((u16)envLight.mWind.mTactWindAngleY);
+    int iVar5 = cM_atan2s(sp18.x, sp18.z);
+    
+    current.angle.y = iVar5;
+    shape_angle.y = iVar5;
+    field_0x29E = 80;
+    speed.x = sp18.x * 40.0f;
+    speed.z = sp18.z * 40.0f;
+    float fVar1 = field_0x29E * 0.5f;
+    current.pos.z = current.pos.z - fVar1 * speed.z;
+    current.pos.x = current.pos.x - fVar1 * speed.x;
+    field_0x29E = field_0x29E + 60;
+    field_0x2A0 = 1.0f;
+    current.pos.y += field_0x2A0 * fVar1 * fVar1 * 0.5f;
+    speed.y = -(field_0x2A0 * fVar1);
 }
 
 /* 00000324-00000388       .text CreateInit__9daWbird_cFv */
-void daWbird_c::CreateInit() {
-    /* Nonmatching */
+BOOL daWbird_c::CreateInit() {
+    tevStr.mRoomNo = current.roomNo;
+    mAction = 3;
+    fopAcM_orderChangeEvent(this, "TACT_WINDOW", 0, -1);
+    field_0x29E = 0;
+    calcMtx();
+    return TRUE;
 }
 
 /* 00000388-000003E0       .text create__9daWbird_cFv */
 cPhs_State daWbird_c::create() {
-    /* Nonmatching */
+    fopAcM_SetupActor(this, daWbird_c);
+    CreateInit();
+    return cPhs_COMPLEATE_e;
 }
 
 /* 000003E0-000003E4       .text actionWait__9daWbird_cFv */
 void daWbird_c::actionWait() {
-    /* Nonmatching */
 }
 
 /* 000003E4-00000474       .text actionEnd__9daWbird_cFv */
 void daWbird_c::actionEnd() {
-    /* Nonmatching */
+    if (dComIfGp_evmng_endCheck(mEventIdx)){
+        dComIfGp_event_reset();
+        fopAcM_delete(this);
+    } else {
+        dComIfGp_evmng_cutEnd(dComIfGp_evmng_getMyStaffId("WINDMAN"));
+    }
 }
 
 /* 00000474-000005DC       .text actionMove__9daWbird_cFv */
 void daWbird_c::actionMove() {
-    /* Nonmatching */
+    daPy_py_c* player = (daPy_py_c*)daPy_getPlayerActorClass();
+    if(field_0x29D){
+        field_0x29D = false;
+        mDoAud_seStart(JA_SE_TAKT_WIND_DEMO);
+    }
+    if (field_0x29E > 0) {
+        field_0x29E--;
+        if (field_0x29E > 60) {
+            fopAcM_posMove(this, NULL);
+            speed.y = speed.y + field_0x2A0;
+        }
+        if (field_0x29E < 15) {
+            dKyw_custom_windpower(0.0f);
+        }
+        if (field_0x29E == 70) {
+            dKyw_tact_wind_set_go();
+        }
+    } else {
+        dComIfGp_evmng_cutEnd(dComIfGp_evmng_getMyStaffId("WINDMAN"));
+        player->changeDemoMoveAngle(mAngle);
+        mAction = 1;
+        mDoAud_seStart(JA_SE_TAKT_WIND_END);
+    }
 }
 
 /* 000005DC-00000850       .text actionSelect__9daWbird_cFv */
 void daWbird_c::actionSelect() {
-    /* Nonmatching */
+    short sVar2 = field_0x29E;
+    if (sVar2 == 10) {
+        dComIfGp_setOperateWindOn();
+        mDoAud_seStart(JA_SE_TAKT_WIND_DISP);
+        field_0x29E++;
+    } else if(sVar2 > 10){
+        mDoAud_seStart(JA_SE_SYS_WTAKT_WIND_AMB);
+        switch (dComIfGp_getOperateWind()) {
+            case 1: {
+                mDoAud_seStart(JA_SE_TAKT_WIND_DECIDE);
+                setStartPos();
+                mAction = 2;
```

Comment:
This actor has an inline in the debug maps that could be implemented here
```suggestion
                setAction(2);
```

Reply:
in the debug map i d_a_wbird.map i cant find this inline? when i replace it with setAction(2); it also doesnt match anymore in objdiff

Reply:
d_a_wbird.map is the non-debug map, it doesn't have any inlines in it. d_a_wbirdD.map is the debug map.

The inline in the header is empty and just a placeholder that I generated with a script, you need to implement it or else it won't do anything.